hyperspy/misc/eds/utils.py

```diff
@@ -13,11 +13,62 @@ def _get_element_and_line(Xray_line):
 
 
 def _get_energy_xray_line(xray_line):
-    energy, line = _get_element_and_line(xray_line)
-    return elements_db[energy]['Atomic_properties']['Xray_lines'][
+    element, line = _get_element_and_line(xray_line)
+    return elements_db[element]['Atomic_properties']['Xray_lines'][
         line]['energy (keV)']
 
 
+def _parse_only_lines(only_lines):
+    if hasattr(only_lines, '__iter__'):
+        if any(isinstance(line, basestring) is False for line in only_lines):
+            return only_lines
+    elif isinstance(only_lines, basestring) is False:
+        return only_lines
+    only_lines = list(only_lines)
+    for only_line in only_lines:
+        if only_line == 'a':
+            only_lines.extend(['Ka', 'La', 'Ma'])
+        elif only_line == 'b':
+            only_lines.extend(['Kb', 'Lb1', 'Mb'])
+    return only_lines
+
+
+def get_xray_lines_near_energy(energy, spread=0.1, only_lines=None):
```

Comment:
Ended up using `width`, while referring to it as a window width in the docstrings, that was also updated.